src/Components/Classes/ClassForm/index.jsx

```diff
@@ -245,7 +245,7 @@ function ClassForm() {
             />
           </div>
         </form>
-        <Button classNameButton="deleteButton" text="Reset" action={reset} />
+        <Button classNameButton="deleteButton" text="RESET" action={reset} />
```

Comment:
this should not be modified
```suggestion
        <Button classNameButton="deleteButton" text="Reset" action={reset} />
```